articles/how-to-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/how-to-build-astatic-site-with-gatsbyjs/
+title: How to build a static site with GatsbyJs
+description:
+author: jethro-magaji
+date: 2020-10-19T00:00:00-08:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-build-astatic-site-with-gatsbyjs/hero.png
+    alt: hero example image computer Node.js
+---
+When I first heard about GatsbyJs, I was confused about what it was, and how I would get started. As time went on, I learned more about it and found out it is awesome. My goal for this article is not to confuse you about GatsbyJs, but to help you get started with building your very first static site using GatsbyJs.
+<!--more-->
+A paradigm shift has begun in the web development world, where static sites are now becoming more of a trend over dynamic sites. Building a dynamic site that required you to set it up and/or maintaining it, is now obsolete.
```

Comment:
It might be a good idea to briefly mention the [JAMStack](https://jamstack.org/what-is-jamstack)

Reply:
> It might be a good idea to briefly mention the [JAMStack](https://jamstack.org/what-is-jamstack)

@Jethro-magaji 
Not sure if we added this suggestion? I didn't see any mention of JAMstack.

Reply:
So should I mention the JAMStack @hectorkambow ?

Reply:
@Jethro-magaji 
Yes - it would add value to the reader to be aware of JAMstack as an alternative out there.

Reply:
Resolved